nrf_modem: update library (#1588)
nrfconnect/sdk-nrfxlib: nrf_modem/doc/dectphy.rst

Afterwards, the application can initialize the DECT PHY interface by calling the :c:func:`nrf_modem_dect_phy_init` function.
Upon successful initialization, both the DECT PHY interface and DECT NR+ physical layer in the modem are ready for operation.

> On nRF9151 devices only, the application can configure band 4 support during initialization by setting the :c:member:`nrf_modem_dect_phy_init_params.band4_support` field to 1.

Reviewer: Updated

.. important::

>    When operating on band 4, the carries outside of the [525, 551] range shall not be used as they interfere with other radio devices, including LTE devices.

Reviewer: Thanks, updated

Afterwards, the application can initialize the DECT PHY interface by calling the :c:func:`nrf_modem_dect_phy_init` function.
Upon successful initialization, both the DECT PHY interface and DECT NR+ physical layer in the modem are ready for operation.

On nRF9151 devices only, the application can configure band 4 support during initialization by setting the :c:member:`nrf_modem_dect_phy_init_params.band4_support` field to 1.

.. note::
   DECT NR+ band 4 is supported by the nRF9151 LACA A0 SiP for R&D evaluation purposes.
   The nRF9151 LACA A0AB SiP will be qualified for DECT NR+ band 4 support.

.. important::
   When operating on band 4, the carries outside of the [525, 551] range shall not be used as they interfere with other radio devices, including LTE devices.

> On nRF9131 devices only, the DECT NR+ PHY firmware locks the calibration data the very first time the DECT PHY interface is initialized.

Reviewer 2: ```suggestion
On nRF9131 Series devices only, the DECT NR+ PHY firmware locks the calibration data the first time the DECT PHY interface is initialized.
```

Reviewer: I have added `Series` but kept `very` to emphasize the fact that this operation happens only once on the device (during its entire lifetime). The interface itself can be initialized multiple times at runtime (e.g. `_init()`, `_deinit()`, `_init()`) and during the device liftime, but only the very first time that `_init()` it is called on the device something is happening.

-----------------

The DECT radio band is divided into different channels, as described in chapter 5.2 of `ETSI TS 103 636-2`_.

The absolute radio channel frequency used for transmission is controlled by the :c:member:`nrf_modem_dect_phy_tx_params.carrier` field on the transmission parameters.

.. note::
   DECT NR+ band 4 is supported by the nRF9151 LACA A0 SiP for R&D evaluation purposes.
   The nRF9151 LACA A0AB SiP will be qualified for DECT NR+ band 4 support.

.. important::
   Follow the local regulations when transmitting.
   To see which frequency channels can be used:
   * In the European Union, you must refer to table 4.3.2.3-2 of `ETSI EN 301 406-2`_.
   * In the USA, you must refer to `47 CFR Part 15.323`_.
   * In all other territories, you must follow the local regulations.

>    When operating on band 4, the carries outside of the [525, 551] range shall not be used as they interfere with other radio devices, including LTE devices.

Reviewer 2: ```suggestion
   When operating on band 4, you must not use the carries outside the range of 525 to 551 as they interfere with other radio devices, including LTE devices.
```

Reviewer: Thanks, updated
